packages/ui-react/src/components/Account/Account.tsx

```diff
@@ -34,73 +33,50 @@ type Props = {
   canUpdateEmail?: boolean;
 };
 
-interface FormErrors {
-  email?: string;
-  confirmationPassword?: string;
-  firstName?: string;
-  lastName?: string;
-  form?: string;
-}
-
 const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }: Props): JSX.Element => {
   const accountController = getModule(AccountController);
 
   const { t } = useTranslation('user');
   const navigate = useNavigate();
   const location = useLocation();
   const [viewPassword, toggleViewPassword] = useToggle();
-  const exportData = useMutation(accountController.exportAccountData);
   const [isAlertVisible, setIsAlertVisible] = useState(false);
+  const exportData = useMutation(accountController.exportAccountData, {
+    onSettled: () => setIsAlertVisible(true),
+  });
   const exportDataMessage = exportData.isSuccess ? t('account.export_data_success') : t('account.export_data_error');
 
-  useEffect(() => {
-    if (exportData.isSuccess || exportData.isError) {
-      setIsAlertVisible(true);
-    }
-  }, [exportData.isSuccess, exportData.isError]);
+  const { data: consentsFields } = useQuery(['consents'], accountController.getConsents);
+  const { data: consentsValues } = useQuery(['consentsValues'], accountController.getConsentsValues);
+  const { data: registrationFields } = useQuery(['registrationFields'], accountController.getRegistrationFields);
```

Comment:
I updated `Registration.tsx` (container) to trigger a fetch from a useEffect, and then take the accountStore as source of truth, but perhaps this (useQuery) is better? I do prefer to use the store as source of truth, but we need a way to trigger the first fetch. 🤔 

Maybe ideally we have a `useAccount` custom hook, that triggers an `accountController.initialize()` (or something similar) and then only use the store as source of truth. But for later, just curious about your thoughts.

Reply:
using `useQuery` was a quick way of getting the data here. The `AccountStore` and `AccountController` are still using `publisherConsents` and `customerConsents`, but I would like to remove these. We could fetch the consents and registration fields on initialize and store it in the AccountStore though...

Reply:
(I thought they were already stored into the AccountStore)

Reply:
Yes, but I wasn't sure we needed them there still